app/models/wpcc/percent.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  class Percent
+    def self.choose_highest(period_percent, user_percent)
```

Comment:
Yes, this object only knows about percents. The two variables are percents and named as percents. On this case I still thinks period_percent makes sense than legal percent because the user story is saying to choose the highest between the legal minimum for this period and the user contribution. Perhaps we should renamed to legal_period_percent? (But can confuse with the other name of the class argh!!)